packages/feedback/src/lib/FeedbackForm.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect } from 'react';
+import { TextField } from '@availity/mui-textfield';
+import { LoadingButton, Button } from '@availity/mui-button';
+import { ToggleButtonGroup, ToggleButton } from '@availity/mui-toggle-button';
+import { Grid, SvgIconProps, ToggleButtonProps, styled } from '@mui/material';
+import { FaceFrownIcon, FaceNeutralIcon, FaceSmileIcon } from '@availity/mui-icon';
+import { avRegionsApi } from '@availity/api-axios';
+import { useForm, SubmitHandler, Controller } from 'react-hook-form';
+
+interface Inputs {
+  feedback: string;
+  smileField: string;
+}
+
+interface SmileButtonProps extends ToggleButtonProps {
+  disabled: boolean;
+  Icon: (props: SvgIconProps) => JSX.Element;
+  label: string;
+  value: string;
+}
+
+interface FeedbackFormProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  analytics: { info: (entries: any) => any };
```

Comment:
`entries` is always an object, right? I would update the type to be `entries: Record<string, unknown>`.  That way we force it to be an object that can have anything

Reply:
fixed